core/templates/dev/head/editor/ExplorationPreview.js

```diff
@@ -59,5 +61,31 @@ oppia.controller('ExplorationPreview', [
     $scope.$on('playerStateChange', function() {
       $scope.allParams = LearnerParamsService.getAllParams();
     });
+
+    explorationData.getData().then($scope._refreshPreviewTab = function() {
```

Comment:
Also, I'm not sure why there's a separate call to getData() here. I think it makes more sense to modify the existing getData() call to do the right thing, though you might still need to pull out the populateExploration() bit into a separate function.